cgmanifest.json

```diff
@@ -2144,19 +2144,19 @@
       "component": {
         "type": "other",
         "other": {
-          "name": "hyperv-daemons",
```

Comment:
Can you check these cgmanifest.json changes? Looks like a number of unrelated packages have changed

Reply:
This is correct. I've ran the script updating the manifest and it put packages in alphabetical order.